src/cmd.rs

```diff
@@ -77,6 +154,165 @@ pub fn osd_pool_set(cluster_handle: rados_t, pool: &str, key: &str, value: &str,
     Ok(())
 }
 
+pub fn osd_set(cluster_handle: rados_t, key: &str, force: bool, simulate: bool) -> Result<(), RadosError> {
+    let cmd = match force {
+        true => {
+            json!({
+                "prefix": "osd set",
+                "key": key,
+                "sure": "--yes-i-really-mean-it",
+            })
+        },
+        false => {
+            json!({
+                "prefix": "osd set",
+                "key": key,
+            })
+        },
+    };
+    debug!("osd set: {:?}", cmd.to_string());
+    if !simulate {
+        ceph_mon_command_without_data(cluster_handle, &cmd.to_string())?;
+    }
+    Ok(())
+}
+
+/// Possible values for the key are:
+/// full|pause|noup|nodown|noout|noin|nobackfill|norebalance|norecover|noscrub|
+/// nodeep-scrub|notieragent
+/// Check src/mon/MonCommands.h in the ceph github repo for more possible
+/// options
+pub fn osd_unset(cluster_handle: rados_t, key: &str, simulate: bool) -> Result<(), RadosError> {
+    let cmd = json!({
+        "prefix": "osd unset",
+        "key": key,
+    });
+    debug!("osd unset: {:?}", cmd.to_string());
+    if !simulate {
+        ceph_mon_command_without_data(cluster_handle, &cmd.to_string())?;
```

Comment:
Instead of constantly creating a JSON object (`let cmd = json!{ ... }`) and then stringifyingn it everywhere, why not make the ceph_mon_command* functions take JSON as their input?

Reply:
Or maybe something like `T: Into<String>`

Reply:
Done